### DISCUSSION TITLE
Strange calibration plot

### DISCUSSION THREAD

**Discussion by MuradOmarov** [CamDavidsonPilon/lifelines#1378] [Q&A]
Hello!
I'm trying to use lifelines in my study. I want to calibrate Cox model (with 10 predictors) with survival_probability_calibration.
There are about 2 000 cases and 130 000 controls in my dataset.
Here my event table from KM for better understanding:
![Снимок экрана 2021-12-30 в 13 14 18](https://user-images.githubusercontent.com/65356761/148075069-15356fdd-0f73-4eb3-a548-c675b0b4a173.png)
And cumulative density:
![Снимок экрана 2021-12-30 в 13 19 39](https://user-images.githubusercontent.com/65356761/148075114-5a2650ba-733e-4c92-ae5a-40236f62e89c.png)
I got C-index 0,77 after fitting. But Calibration curves look very strange:
![Снимок экрана 2021-12-30 в 13 18 08](https://user-images.githubusercontent.com/65356761/148075182-b79f39b7-e804-4840-b7d9-24130062ec8f.png)
![Снимок экрана 2021-12-30 в 13 18 42](https://user-images.githubusercontent.com/65356761/148075192-701055c7-5508-45d6-85ad-e45dbf1f1f41.png)
Can you tell me what I'm doing wrong?
Thanks in advance

**Comment by CamDavidsonPilon** [comment]
Hi there, I don't think you are doing anything wrong. The error is likely in the calibration part - there may just be too few cases to build a reliable calibration model. Are you able to share with me the dataset? Publically, here, or privately, at cam.davidson.pilon@gmail.com

**Reply by MuradOmarov** [reply]
Thanks for your reply! 
I sent the data to the e-mail.
